reflectapi/Cargo.toml

```diff
@@ -53,6 +53,7 @@ check_keyword = { version = "0.2.0", optional = true }
 
 # optional 3rd party dependencies for client runtime
 reqwest = { version = "0.12", optional = true }
+reqwest-middleware = { version = "0.3", optional = true }
```

Comment:
Don't you have to add this as a feature as well?

I assume the clients will have to depend on `reqwest-tracing`, and create their custom reqwest client - might be worth updating the demo to show how to use it.

Reply:
Yep pretty much. Can do. I think it implicitly creates a feature under the same name. Not sure if that method is deprecated or not.